assets/i18n/en.json

```diff
@@ -139,7 +147,7 @@
     "I18N_BLOG_POST_EDITOR_TITLE_MAX_LENGTH_ERROR": "Blog Post title should have at most <[maxChars]> characters.",
     "I18N_BLOG_POST_EDITOR_TITLE_MIN_LENGTH_ERROR": "Blog Post title should have at least <[minChars]> characters.",
     "I18N_BLOG_POST_PAGE_RECOMMENDATION_SECTON_HEADING": "Suggested For You.",
-    "I18N_BLOG_POST_PAGE_TAGS_HEADING": "Tags",
+    "I18N_BLOG_POST_PAGE_TAGS_HEADING": "Tags:",
```

Comment:
The colon should be in the code, not here.

Reply:
Done